documentation/existing.md

```diff
@@ -21,7 +27,7 @@ Make sure Node VM names must comply with the regex `[a-z](([-0-9a-z]+)?[0-9a-z])
   * VM names can not have capital letters, any special characters except `.` and `-`.
```

Comment:
cannot is a single word. Can you change it all places.

Reply:
Done